packages/circuit-ui/hooks/useFocusList/useFocusList.stories.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2021, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable jsx-a11y/no-redundant-roles */
```

Comment:
Why do we need to add the list role explicitly? Is it just for the example?

Reply:
Safari has a ~bug~feature where it removes list semantics when using `list-style: none` (see ["Fixing" Lists](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)  by Scott O'Hara). Explicitly setting the `role` attribute fixes this.

Reply:
A big 👏 for Safari